[Key Vault - Certificates]  Certificates client - create, read, update and delete > can read, cancel and delete a certificate's operation failing in nightly runs

Key Vault - Certificates nightly test runs are failing with:

> Error message:
> expected 'Pending Certificate not found: crudce…' to deeply equal 'Pending certificate not found: crudce…'
> 
> 
> 
> Stack trace:
> AssertionError: expected 'Pending Certificate not found: crudce…' to deeply equal 'Pending certificate not found: crudce…'
> 
> Expected: "Pending certificate not found: crudcertoperation-6645"
> Received: "Pending Certificate not found: crudcertoperation-6645"
> 
>  ❯ test/public/CRUD.spec.ts:475:27

For more details check here:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4311920&view=results

--- a/sdk/keyvault/keyvault-certificates/test/public/CRUD.spec.ts
+++ b/sdk/keyvault/keyvault-certificates/test/public/CRUD.spec.ts
@@ -438,7 +438,7 @@ describe("Certificates client - create, read, update and delete", () => {
     expect(updated.policy!.issuerName).toEqual("Self");
   });
 
-  it("can read, cancel and delete a certificate's operation", { retry: 5 }, async function () {
+  it("can read, cancel and delete a certificate's operation", async function () {
     const certificateName = recorder.variable(
       "crudcertoperation",
       `crudcertoperation-${Math.floor(Math.random() * 10000)}`,
@@ -465,14 +465,18 @@ describe("Certificates client - create, read, update and delete", () => {
     // Delete
     await client.deleteCertificateOperation(certificateName);
 
-    let error;
+    let errorMessage: string;
     try {
       await client.getCertificateOperation(certificateName);
       throw Error("Expecting an error but not catching one.");
     } catch (e: any) {
-      error = e;
+      errorMessage = e.message;
     }
-    expect(error.message).toEqual(`Pending certificate not found: ${certificateName}`);
+    let expectedErrorMessage = new RegExp(
+      `pending certificate not found: ${certificateName}`,
+      /* case insensitive */ "i",
+    );
+    expect(errorMessage).toMatch(expectedErrorMessage);
   });
 
   it("can set, read and delete a certificate's contacts", async function () {

--- a/sdk/keyvault/keyvault-certificates/test/public/CRUD.spec.ts
+++ b/sdk/keyvault/keyvault-certificates/test/public/CRUD.spec.ts
@@ -472,7 +472,7 @@ describe("Certificates client - create, read, update and delete", () => {
     } catch (e: any) {
       errorMessage = e.message;
     }
-    let expectedErrorMessage = new RegExp(
+    const expectedErrorMessage = new RegExp(
       `pending certificate not found: ${certificateName}`,
       /* case insensitive */ "i",
     );

